### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by GustavoRSSilva**
The Master is behind developer and the documentantion does not reflect what it is there

### CODE REVIEW

File: deploy/test/AppCoinsIAB.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+var AppCoinsIAB = artifacts.require("./AppCoinsIAB.sol");
```
Comment (DioPires) [inline deploy/test/AppCoinsIAB.js]: #### Tests:
```diff 
+ 32 passing
```
#### Test coverage: 
82.59 %

Comment (DioPires) [inline deploy/test/AppCoinsIAB.js]: #### Tests:
```diff 
+ 32 passing
```
#### Test coverage: 
82.59 %

**Review by tcrosado — COMMENTED** [review]
As dev already has Advertisement storage, i think we should first fix the appc transfer when only Advertisement contract is upgraded.
A secondary issue we should discuss is how frequently we should merge from dev to master, i suggest each time we deploy to main net.

**Comment by GustavoRSSilva** [comment]
@tcrosado good point, we should wait to upgrade the contracts on mainnet to reflect this changes before merging with dev.

**Pull request closed without merging**